draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -426,9 +427,12 @@ Exemple with:
 2. cmac: 0x4E822D9775B2649928F82066AF804FEC
 3. IID: 0x28F82066AF804FEC
 ~~~~
-
 {: #Fig-iid-computation-example title='Example of IID computation.'}
 
+There is a small probability of IID collision in a network, if such event occurs
+the IID can be changed by rekeying the device on L2 level (ie: LoRaWAN rejoin).
```

Comment:
I would not say "rejoin" as it has a specific meaning in LoRaWAN 1.1.x. 
"LoRaWAN new join" ?

Reply:
What about `(ie: trigger a LoRaWAN join)` instead ?

Reply:
> What about `(ie: trigger a LoRaWAN join)` instead ?

Perfect!